### DISCUSSION TITLE
Spotkanie "Architektura w Azure pod CRUD" #azure-solutions-architect

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#48] [Ogólna]
Cześć 

Zapraszamy wszystkich chętnych do **dyskusji związanej z architekturą w Azure**. 
Tematem spotkania jest architektura dla CRUD z naciskiem na infrastrukturę.
Termin: **17/09 20:00**

Na spotkaniu robimy burzę mózgów dotyczącą zadania stworzonego przez @kaluzaaa 
_Zaprojektuj infrastukturę w Azure pod CRUD w technologii w której na codzień pracujesz.
Tylko jeden region.
Backup danych 14 dni.
Zaprojektuj dwie wersje:
tanio
tanio, ale zabezpieczonej tylko wejściem z Twojej sieci (vnet, vpn)
Policz koszt miesięczny dla obu rozwiązań._

Link do Teams: https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmY3MzE0NzEtYjU3NC00NzkxLTliY2YtMTNlMTliMDUwNWI4%40thread.v2/0?context=%7b%22Tid%22%3a%22ef60a5a4-900c-4789-89cd-0c42f28ef541%22%2c%22Oid%22%3a%22843f36ea-2af5-4be3-a168-6b8fd3acd1b3%22%7d

Link do kalendarza: https://evt.mx/lm7HeGPy

**Comment by maciejgos** [comment]
Zapraszamy osoby z dowolnym poziomem wiedzy, oraz fajnie by było jakby ktoś przygotował swoją propozycje rozwiązania 😉

**Comment by mgrabarz** [comment]
Ja proponuje, by jeszcze tutaj jakiegoś [ICSa](https://pl.wikipedia.org/wiki/ICalendar) wrzucić, to się ludziom wpisze do kalendarzy :)

**Reply by maciejgos** [reply]
Pracujemy nad tym :)

**Reply by aryll** [reply]
I rozpracowaliśmy :)
